src/xrpld/rpc/detail/RPCCall.cpp

```diff
@@ -459,7 +528,12 @@ class RPCParser
         Json::Value jvRequest(Json::objectValue);
 
         if (jvParams.size())
-            jvRequest[jss::min_count] = jvParams[0u].asUInt();
+        {
+            if (auto minCount = jvParseUInt(jvParams[0u]))
```

Comment:
My bad, I missed this:

```suggestion
            if (auto const minCount = jvParseUInt(jvParams[0u]))
```